packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -1005,6 +1028,12 @@ const TagBox = (SelectBox as any).inherit({
     return selectedItems;
   },
 
+  _processDataSourceChanging() {
+    this._newDataSource = true;
```

Comment:
```suggestion
    this._isDataSourceChanging = true;
```

Reply:
actually it's already not changing at this moment

Reply:
renamed to _isDataSourceOptionChanged